src/App.tsx

```diff
@@ -7,38 +7,125 @@ import classNames from 'classnames';
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
+import { client } from './utils/fetchClient';
+import { User } from './types/User';
 import { Loader } from './components/Loader';
+import { Post } from './types/Post';
 
 export const App: React.FC = () => {
+  const [users, setUsers] = useState<User[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [selectedUserId, setSelectedUserId] = useState(0);
+  const [visiblePosts, setVisiblePosts] = useState<Post[]>([]);
+  const [selectedPost, setSelectedPost] = useState<Post | null>(null);
+  const [hasError, setHasError] = useState(false);
+  const isSidebar = selectedPost && true;
+
+  const loadUsers = (url: string) : Promise<any> => {
```

Comment:
```suggestion
  const loadUsers = (url: string): Promise<****> => {
```
do could we add type instead of any?